### DISCUSSION TITLE
Standardize how to contribute

### DISCUSSION THREAD

**Discussion by StepKie** [enisn/UraniumUI#559] [General]
The[ contribution guide](https://github.com/enisn/UraniumUI/blob/develop/CONTRIBUTING.md) is very much appreciated.
There are however a couple of open questions that were asked in a ticket [here](https://github.com/enisn/UraniumUI/issues/252#issuecomment-1895344489), but I am creating a discussion item because maybe it is better suited here.

1. what are the settings for formatting and warnings? there are for example source files indented with tabs among those indented with spaces. I would suggest adding an .editorconfig file, and then ensuring that warnings are always at 0 - currently, I get a significant amout of warnings / messages (see screenshot)
2. such an .editorconfig would ensure that PRs would (at least formally) adhere to the repo maintainers coding standards.
3. what are issues that are "up for grabs" to anyone, i.e. that you would be happy to accept PRs for? E.g., is it all tickets that are open and tagged "help wanted"? can I easily filter them/see them in a single place somewhere?

<img width="840" alt="image" src="https://github.com/enisn/UraniumUI/assets/17760200/429f2b84-fb7a-41d1-bed8-6b5ac02e3ceb">
<img width="508" alt="image" src="https://github.com/enisn/UraniumUI/assets/17760200/f8191ddc-e4f5-490f-9be8-6c9d64caae5f">

**Comment by StepKie** [comment]
Edit: this thread is mainly a question for @enisn.

**Reply by enisn** [reply]
Thanks for your proposal. All of them seem right. I'll try to update when possible

**Reply by StepKie** [reply]
@enisn Would you accept a PR for this?

i.e., can I convert this into an issue and submit a PR for it (with and .editorconfig, etc.)?
